.jenkins/pipelines/Azure/Linux/Jenkinsfile

```diff
@@ -169,15 +169,15 @@ def ACCHostVerificationTest(String version, String build_type) {
                 def task = """
                            cmake ${WORKSPACE} -G Ninja -DHAS_QUOTE_PROVIDER=ON -DCMAKE_BUILD_TYPE=${build_type} -Wdev
                            ninja -v
-                           pushd tests/host_verify/host
+                           pushd tests/tools/oecert
```

Comment:
The following task has to happen in host_verify/host. Do not go to tests/tools/oecert. Provide full path to enclave file in oecert/host/host.c instead.

Reply:
Change to full path.